cli2telegram/config.py

```diff
@@ -73,3 +73,9 @@ def __new__(cls, *args, **kwargs):
         description="Time interval after which the retry should be cancelled.",
         default="1h",
     )
+    DAEMON_PIPE_PATH = StringConfigEntry(
+        key_path=[KEY_ROOT, "daemon_pipe_path"],
+        description="Unix named pipe path.",
+        default="/tmp/cli2telegram-",
```

Comment:
Is the `-` important? What does it do?

Reply:
It serves no function, only a default. I had in mind the `/tmp/cli2telegram-someRandomString` naming scheme used by a number of linux daemons.
![image](https://user-images.githubusercontent.com/3334013/116164056-e36c2300-a6ad-11eb-9ea8-17f73508ccbc.png)

Reply:
But there is nothing appended to this value at runtime, right? So either the default has to already be a generated value, which would change for every process instance, or we just use some static default path. Since the path is configurable anyway I don't think we have to use a generated path here and can just use `/tmp/cli2telegram-pipe` or something similar as a default.